src/lib.rs

```diff
@@ -238,10 +238,10 @@ pub fn middleware(app: Arc<App>) -> MiddlewareBuilder {
     // Sets the current user on each request.
     m.add(user::Middleware);
 
-    // Serve the static files in the *dist* directory, which are the frontend assets.
-    // Not needed for the backend tests.
+    // Serve the static files in the *local_uploads* directory, which contains locally uploaded
+    // crates. Not needed for the backend tests.
     if env != Env::Test {
-        m.around(dist::Middleware::default());
+        m.around(local_uploads::Middleware::default());
```

Comment:
Wouldn't this break prod ?
It seems that the `dist::Middleware` is used to serve the Ember application, styles & other assets. Does it still work after this change ?